src/entity/MarketingCta.ts

```diff
@@ -0,0 +1,28 @@
+import { Column, Entity, PrimaryColumn } from 'typeorm';
+
+export type MarketingCtaVariant = 'card' | 'popover';
+
+export type MarketingCtaFlags = Partial<{
+  tagText: string;
+  tagColor: string;
+  ctaUrl: string;
+  ctaText: string;
+  description: string;
+  image: string;
+  title: string;
+}>;
+
+@Entity()
+export class MarketingCta {
+  @PrimaryColumn({ type: 'text' })
+  campaignId?: string;
```

Comment:
@capJavert I was thinking, as this is going to be unique, we don't need a specific ID row.
What do you think?

Reply:
If that is the case I agree.

Reply:
In that case it shouldn't be optional right :)

Reply:
Yeah, it shouldn't. Fixed it.